### PR TITLE
Add 2024 Q4 Sigstore Update

### PR DESCRIPTION
**Pull request by haydentherapper**
Closes https://github.com/ossf/tac/issues/412

### CODE REVIEW

**Review by steiza — APPROVED** [review]
Thanks!

**Review by bobcallaway — APPROVED** [review]

**Review by mlieberman85 — APPROVED** [review]
LGTM

**Comment by haydentherapper** [comment]
Thanks all, can someone merge?

**Review by marcelamelara — APPROVED** [review]
Great update, thanks @haydentherapper !

**Comment by marcelamelara** [comment]
> Thanks all, can someone merge?

These types of PRs need 5 approvals, I'll ping folks on Slack.

File: TI-reports/2024/2024-Q4-Sigstore.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# 2024 Q4 Sigstore
+
+## Overview
+
+### Conference
+
+Our second [SigstoreCon: Supply Chain Day](https://events.linuxfoundation.org/sigstorecon-supply-chain-day/) conference just wrapped up. With just over 90 attendees, SigstoreCon brought together individuals and organizations excited about not only Sigstore but other supply chain initiatives such as SLSA, SBOM, or in-toto. Talks are recorded [here](https://www.youtube.com/playlist?list=PLM6mY5TOhY1E02_fQWqfQk_gMRHHtX0q6).
```
Comment (lehors) [inline TI-reports/2024/2024-Q4-Sigstore.md:7]: Using “here” as a link anchor is an anti-pattern.
For more info see the following blog post (from a renowned expert. ;-) : https://lehors.wordpress.com/2009/01/29/linking-the-proper-way/

**Review by lehors — APPROVED** [review]
Sigstore has all the signs of a successful, well run project which should inspire other OpenSSF projects!

Although I think it should, our report template doesn't currently prompt for this but I'd appreciate any info on how Sigstore is doing with the implementation of the Security Baseline. What's the status? Are there any issues encountered?

Thanks.

**Comment by lehors** [comment]
> > Thanks all, can someone merge?
> 
> These types of PRs need 5 approvals, I'll ping folks on Slack.

I actually think that TI reports should get reviewed by as many TAC members as possible. There is no rush and in my opinion it's more important that everybody gets a chance to be informed.

**Comment by bobcallaway** [comment]
> Although I think it should, our report template doesn't currently prompt for this but I'd appreciate any info on how Sigstore is doing with the implementation of the Security Baseline. What's the status? Are there any issues encountered?

As part of the TI graduation process earlier in 2024, the Sigstore TSC documented [status against the best practices badge requirements](https://github.com/sigstore/TSC/blob/c6b22e2c2e65d032725338ca5964c44502fc9ac8/docs/best_practices_badge.md) - which I know isn't exactly the same as the security baseline, but is probably the closest we have to documenting the overall posture of the Sigstore project.

**Review by sevansdell — APPROVED** [review]
Great work Sigstore!

I'm also personally interested in the ability to sign model cards, with an effort in the AIML WG, Model Signing SIG.

**Pull request merged**